gpytorch/gpytorchopt/gpytorchopt/acquisition/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
```

Comment:
Should we maintain py2 compatibility as long as pytorch does? If so, we need to convert the typing into comment-style type hints. More generally, we should add types / type hints throughout the code early on, as this will simplify development.

I would actually be fine restricting the bayesopt functionality to py3 only...

@jacobrgardner, @gpleiss, what's your take on this?